controllers/managedocs_controller.go

```diff
@@ -913,6 +924,67 @@ func (r *ManagedOCSReconciler) reconcileAlertRelabelConfigSecret() error {
 	return nil
 }
 
+func (r *ManagedOCSReconciler) reconcileKubeRbacConfigMap() error {
+	r.Log.Info("Reconciling kubeRbacConfigMap")
+
+	_, err := ctrl.CreateOrUpdate(r.ctx, r.Client, r.kubeRbacConfigMap, func() error {
+		if err := r.own(r.kubeRbacConfigMap); err != nil {
+			return err
+		}
+
+		kubeRbacConfig := struct {
+			Authorization struct {
+				Static []struct {
+					Path            string `yaml:"path"`
+					ResourceRequest bool   `yaml:"resourceRequest"`
+					Verb            string `yaml:"verb"`
+				} `yaml:"static"`
+			} `yaml:"authorization"`
+		}{
+			Authorization: struct {
+				Static []struct {
+					Path            string "yaml:\"path\""
+					ResourceRequest bool   "yaml:\"resourceRequest\""
+					Verb            string "yaml:\"verb\""
+				} "yaml:\"static\""
+			}{
+				Static: []struct {
+					Path            string "yaml:\"path\""
+					ResourceRequest bool   "yaml:\"resourceRequest\""
+					Verb            string "yaml:\"verb\""
+				}{
+					{
+						Path:            "/metrics",
+						ResourceRequest: false,
+						Verb:            "get",
+					},
+					{
+						Path:            "/federate",
+						ResourceRequest: false,
+						Verb:            "get",
+					},
+				},
+			},
+		}
+
+		config, err := yaml.Marshal(kubeRbacConfig)
+		if err != nil {
+			return fmt.Errorf("unable to encode kubeRbacConfig: %v", err)
+		}
+
+		r.kubeRbacConfigMap.Data = map[string]string{
+			"config-file.yaml": string(config),
+		}
+		return nil
+	})
+
+	if err != nil {
+		return fmt.Errorf("unable to create kubeRbacConfig config map: %v", err)
+	}
+
+	return nil
+}
+
 // reconcilePrometheusService function wait for prometheus Service
 // to start and sets appropriate annotation for 'service-ca' controller
 func (r *ManagedOCSReconciler) reconcilePrometheusService() error { //nolint:deadcode,unused
```

Comment:
- the nolint directives can be removed now i think
- pls do the same in remaining places

Reply:
Yes, I missed to remove these